### PR TITLE
RD2: Enhanced Recurring Donations label updates

### PR DESCRIPTION
**Pull request by sam-knox**
We made several changes to the Update to the Enhanced Recurring Donations page, and a couple of field labels.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by sam-knox** [comment]
**lurch: attach W-037832

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HpK3QAK" target="_blank">W-037832</a>

**Comment by sam-knox** [comment]
@RadSFDeveloper - This PR is ready for review. I tried to use encoded HTML on the splash page that checks if CRLPs and LEX are enabled but it didn't work. If there's a way, let me know.

Also, I updated the text for RD2_EnablementMigrationCompleteMessage, but I didn't see that message after I went through all of the enablement steps including the final migration.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -1548,15 +1548,15 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
...
+        <value>Almost there! Just need you to check on a couple of things first . . .</value>
     </labels>
     <labels>
         <fullName>RD2_EnablementDisabledMessage</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Enablement page message displayed due to missing prerequisites</shortDescription>
-        <value>You must enable both Customizable Rollups and Lightning Experience in order to enable Enhanced Recurring Donations.</value>
+        <value>(1) If you're not already using Lightning Experience, click Switch to Lightning Experience at the top of your screen. (2) Enable Customizable Rollups by going to NPSP Settings|Donations|Customizable Rollups. Then return here to the Upgrade to Enhanced Recurring Donations page.</value>
```
Comment (RadSFDeveloper) [inline src/labels/CustomLabels.labels]: The message is rendered as is. I have committed a change to render text as `lightning:formattedRichText` and added a new line to separate instructions. Note, the text is center aligned, let me know if it needs to change.
Screen Shot 2019-12-31 at 4 15 14 PM

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -1732,7 +1734,13 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Message displayed on a successful migration</shortDescription>
-        <value>Your migration was successful with no errors. All set up steps are now complete. You're ready to begin using Enhanced Recurring Donations.</value>
+        <value>You're now finished with the Enhanced Recurring Donations upgrade process. You've enabled the new functionality, installed metadata, and migrated your existing data.
```
Comment (RadSFDeveloper) [inline src/labels/CustomLabels.labels]: The message was not displayed, the issue is fixed now. Can you please verify?

Comment (sam-knox) [reply src/labels/CustomLabels.labels]: @RadSFDeveloper - Yes, I see the message now, thank you.

**Comment by sam-knox** [comment]
@ravensisco - This PR is ready for QA. Thanks!

**Pull request merged**